modules/virt-cloud-init-fields-web.adoc

```diff
@@ -19,3 +19,8 @@
 |Custom script
 |Replaces other options with a field in which you paste a custom cloud-init script.
 |===
+
+[NOTE]
+====
+Statically inject an SSH key using the custom script in Cloud-init or in the wizard. This allows you to securely and remotely manage virtual machines and manage and transfer information. This step is strongly recommended to secure your VM. 
```

Comment:
The note says "This step is strongly recommended," so I wonder if it actually belongs in [the procedure](https://deploy-preview-33086--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-creating-template-wizard-web_virt-creating-vm-template) under step 8.

Nitpick 1: I think it's generally "cloud-init" without the capital C, unless it's the beginning of a sentence. (If it's referring to a UI element titled "Cloud-init," the text should be bold.)

Nitpick 2: s/using the custom script/by using the custom script